pkgs/top-level/python-packages.nix

```diff
@@ -2220,7 +2220,8 @@ in {
 
   pyfribidi = callPackage ../development/python-modules/pyfribidi { };
 
-  docker_compose = callPackage ../development/python-modules/docker_compose {};
+  docker-compose = callPackage ../development/python-modules/docker_compose {};
+  docker_compose = self.docker-compose; # legacy alias
```

Comment:
Please don't add aliases. They only make it more confusing when overriding stuff. Because users eventually will have to update their expressions, and because an alias like this doesn't cause any notification of it, I don't see the point of it.

Reply:
I moved it out of python-packages.nix since it is not a library.